README.md

```diff
@@ -140,7 +140,9 @@ To install, download the `haskell-language-server-wrapper` executable for your p
 
 ### Arch Linux
 
-If you are using Arch Linux with **dynamically linked** Haskell packages from `pacman`,
+The preferred method of installation for development purposes is to use the [haskell-language-server-bin](https://aur.archlinux.org/packages/haskell-language-server-bin) package from AUR. This package contains statically linked binaries for each supported GHC version and `haskell-language-server-wrapper` for dynamic GHC version selection. It is updated regularly, requires no additional dependencies, and is independent of other haskell packages you may have on your system, including GHC. Its size is relatively large (approx. 900 MB), but if this is a problem for you, during installation you can disable the GHC versions you will not be using by editing the PKGBUILD file.
```

Comment:
Please break this single-line paragraph into several lines, now it's too long!

Reply:
Sorry for the delay. Fixed in 203da29